static/templates/settings/organization_permissions_admin.hbs

```diff
@@ -32,7 +32,7 @@
                 <div class="dependent-block">
                     <p id="allowed_domains_label" class="inline-block"></p>
                     {{#if is_admin }}
-                    <a data-toggle="modal" href="#realm_domains_modal">{{t "[Configure]" }}</a>
+                    <button type="button" id="show_realm_domains_modal" class="button save-discard-widget-button">{{t "[Configure]" }}</button>
```

Comment:
Any reason to do this change? This currentlydoes not look good, and should not have `save-discard-widget-button` class.
![Screenshot from 2021-11-08 13-00-10](https://user-images.githubusercontent.com/35494118/140701060-dcb137e0-4972-4c18-8a63-ddb46c0edb10.png)

Reply:
Bootstrap modals can be invoked by `a` tags, but Micromodal modals can't. So I changed it to a button and added a click handler.
I added the `save-discard-widget-button` class to make the button look like the rest

Reply:
I tested it, we can open micromodal by just using click-handler in jquery instead of href attribute in the template. You can just update the `a` tag to be `<a id="show_realm_domains_modal" role="button">{{t "[Configure]" }}</a>` like we have for `Disable` button for `New stream notifications` dropdown list widget in `Organization settings`.

Reply:
Ah, Nice. Fixed. Thanks.